charts/simple-app/Chart.yaml

```diff
@@ -2,13 +2,15 @@ apiVersion: v2
 name: simple-app
 description: Default Microservice Helm Chart
 type: application
-version: 0.15.3
+version: 0.16.0
 appVersion: latest
 maintainers:
   - name: diranged
     email: matt@nextdoor.com
 dependencies:
+
   - name: istio-alerts
-    version: 0.1.1
-    repository: "file://../istio-alerts/"
-    condition: istio-alerts.enabled
+    alias: istioRules
+    version: 0.1.2
+    repository: https://k8s-charts.nextdoor.com
```

Comment:
By using the user-facing repository endpoint, we no longer are forced to make a change  to all of the nested charts that depend on `istio-alerts` each time we update `istio-alerts`. Instead, each chart can be upgraded independently when we're ready. That seems better to me and less confusing.